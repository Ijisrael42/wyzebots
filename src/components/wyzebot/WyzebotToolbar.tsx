import { Box, Button, } from '@material-ui/core';
import { Link as RouterLink } from 'react-router-dom';

const WyzebotToolbar: React.FC<any> = (props) => (
  <Box {...props}>
    <Box sx={{ display: 'flex', justifyContent: 'flex-end' }} >
      {/* <Button> Import </Button>  <Button sx={{ mx: 1 }}> Export </Button> */}
      <Button component={RouterLink}  to={`/${props.module}/create`} color="primary" variant="contained" > Add Wyzebot </Button>
    </Box>    
  </Box>
);

export default WyzebotToolbar;
