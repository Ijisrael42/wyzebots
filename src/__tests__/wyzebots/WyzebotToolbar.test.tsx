import React from 'react';
import { shallow } from 'enzyme';
import WyzebotToolbar from '../../components/wyzebot/WyzebotToolbar';

test('renders WyzebotToolbar', () => {
  const component = shallow(<WyzebotToolbar module="wyzebots" />); 
  expect(component).toMatchSnapshot();
});