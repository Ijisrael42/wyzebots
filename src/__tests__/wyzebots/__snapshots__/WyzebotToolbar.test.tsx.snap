// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders WyzebotToolbar 1`] = `
<ForwardRef(Box)
  module="wyzebots"
>
  <ForwardRef(Box)
    sx={
      Object {
        "display": "flex",
        "justifyContent": "flex-end",
      }
    }
  >
    <ForwardRef(Button)
      color="primary"
      component={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        }
      }
      to="/wyzebots/create"
      variant="contained"
    >
       Add Wyzebot 
    </ForwardRef(Button)>
  </ForwardRef(Box)>
</ForwardRef(Box)>
`;
